src/bundle/Resources/views/themes/standard/design_system/components/tag.html.twig

```diff
@@ -0,0 +1,52 @@
+{% set ghost_tag = type is defined and 'ghost' in type %}
```

Comment:
If you implement getter for `is_ghost_type` you'll be able to remove this line and use `is_ghost_type` coming directly from variables